lab-davidw/README.md

```diff
@@ -0,0 +1,8 @@
+# Lab-03-fs-readfile
```

Comment:
Nice job including a read me. Generally, in your read me, you would want to discuss how to run your app, i.e commands, specific npm packages.